lasagne/utils.py

```diff
@@ -305,30 +306,50 @@ def create_param(spec, shape, name=None):
     name : string, optional
         If a new variable is created, the name to give to the parameter
         variable. This is ignored if `spec` is already a Theano expression
-        or shared variable.
+        or shared variable and which already has a name.
 
     Returns
     -------
     Theano shared variable or Theano expression
         A Theano shared variable or expression representing layer parameters.
-        If a numpy array was provided, a shared variable is initialized to
-        contain this array. If a shared variable or expression was provided,
-        it is simply returned. If a callable was provided, it is called, and
-        its output is used to initialize a shared variable.
+        If an scalar or a numpy array was provided, a shared variable is
+        initialized to contain this array. If a shared variable or expression
+        was provided, it is simply returned. If a callable was provided, it is
+        called, and its output is used to initialize a shared variable.
 
     Notes
     -----
     This function is called by :meth:`Layer.add_param()` in the constructor
     of most :class:`Layer` subclasses. This enables those layers to
-    support initialization with numpy arrays, existing Theano shared variables
-    or expressions, and callables for generating initial parameter values.
+    support initialization with scalars, numpy arrays, existing Theano shared
+    variables or expressions, and callables for generating initial parameter
+    values, Theano expressions, or shared variables.
     """
+    # to check if argument is a number
+    import numbers
```

Comment:
If you're fixing the spelling above, could you make this a line comment?
```python
import numbers  # to check if argument is a scalar number
```
Otherwise it looks as if this comment applies to the whole following code block.